### PR TITLE
WinUI 890011 - Images added for Scheduler tooltip and cell padding

### PR DESCRIPTION
**Pull request by TamilarasanGunasekaran**
I have added images and GIF files for the Scheduler tooltip and cell padding support.

### CODE REVIEW

File: winui/Scheduler/Day-Week-Views.md [hunk 1]
```diff
@@ -516,5 +516,7 @@ this.Schedule.DaysViewSettings.CellRightPadding = 30;
 {% endhighlight %}
 {% endtabs %}
 
+![cell-right-padding-support-in-winui-scheduler-timeslot-views](DayandWeekViews_Images/cell-padding-in-winui-scheduler.png)
```
Comment (Karthickmani97) [inline winui/Scheduler/Day-Week-Views.md:519]: Change image name to `cell-right-padding-support-in-WPF-scheduler`

change in all places.

[the remark this answers is not in the document]
Comment (TamilarasanGunasekaran) [reply winui/Scheduler/Day-Week-Views.md]: Replaced the images.

File: winui/Scheduler/Appointments.md [hunk 2]
```diff
@@ -775,6 +775,9 @@ this.Schedule.EnableToolTip = true;
 {% endhighlight %}
 {% endtabs %}
 
+![appointment-tool-tip-support-in-winui-scheduler](Appointment_Images/tooltip-winui-scheduler.gif)
```
[the remark this answers is not in the document]
Comment (TamilarasanGunasekaran) [reply winui/Scheduler/Appointments.md]: Changed.

File: winui/Scheduler/Day-Week-Views.md (hunk 1 above)
Comment (TamilarasanGunasekaran) [reply winui/Scheduler/Day-Week-Views.md]: Changed.

**Review by Karthickmani97 — APPROVED** [review]
This is fine.

**Pull request merged**